### PR TITLE
Book detail iii

### PR DESCRIPTION
**Pull request by Juliocho990**
## Summary
- Card component was created
- Some border styles were added

## Screenshot
book-detail-III

## Trello card
https://trello.com/c/dU7B4tvN/28-detalle-de-libro-iii-estructura

### CODE REVIEW

**Review by CrisVega08 — CHANGES_REQUESTED** [review]
Same about branch

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "compile-hero.disable-compile-files-on-did-save-code": true
+}
```
Comment (eledwinn) [inline .vscode/settings.json]: Empty line at EOF.

File: scss/detail.scss [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  min-height: 100vh;
+  min-width: 100vw;
+}
+
 .card {
+  background-color: white;
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
```
Comment (eledwinn) [inline scss/detail.scss]: Try with `rgba($black, 0.5);`

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (CrisVega08) [reply .vscode/settings.json]: I think you can add this to .gitignore

File: scss/detail.scss (hunk 2 above)
Comment (Juliocho990) [reply scss/detail.scss]: It works, thank you!

File: .vscode/settings.json (hunk 1 above)
Comment (Juliocho990) [reply .vscode/settings.json]: I've added it, thank you.

File: .gitignore [hunk 3]
```diff
@@ -8,3 +8,7 @@
 
 ## System ##
 .DS_Store
+
+
+## Visual Studio Code ##
+.vscode/
```
Comment (eledwinn) [inline .gitignore]: Add empty at EOF

File: scss/detail.scss [hunk 4]
```diff
@@ -1,4 +1,16 @@
+.container {
+  align-items: center;
+  background: $white-smoke;
+  display: flex;
+  flex-direction: row;
```
Comment (eledwinn) [inline scss/detail.scss]: Default flex direction is row, therefore, it is not necessary to add it.

**Review by eledwinn — APPROVED** [review]

File: scss/detail.scss (hunk 4 above)
Comment (Juliocho990) [reply scss/detail.scss]: Thanks, it was something I didn't know, I'll try to keep it in mind

File: .gitignore (hunk 3 above)
Comment (Juliocho990) [reply .gitignore]: ok

File: .vscode/settings.json (hunk 1 above)
Comment (CrisVega08) [reply .vscode/settings.json]: @Juliocho990  The file continue appearing

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 5]
```diff
@@ -1,4 +1,15 @@
...
+  align-items: center;
+  background: $white-smoke;
+  display: flex;
+  justify-content: center;
+  min-height: 100vh;
+  min-width: 100vw;
+}
+
 .card {
+  background-color: white;
```
Comment (CrisVega08) [inline scss/detail.scss]: Use variable here

**Review by CrisVega08 — CHANGES_REQUESTED** [review]
@Juliocho990 one comment more

File: scss/detail.scss (hunk 5 above)
Comment (Juliocho990) [reply scss/detail.scss]: ok

**Review by eledwinn — APPROVED** [review]

**Pull request merged**